INSTALL.rst

```diff
@@ -248,3 +248,19 @@ ISAAC
 - *Spack:* ``spack install isaac``
 - *from source:* build the *in situ library* and its dependencies as described in `ISAAC's INSTALL.md <https://github.com/ComputationalRadiationPhysics/isaac/blob/master/INSTALL.md>`_
 - *environment:* set environment variable ``CMAKE_PREFIX_PATH`` for each dependency and the ISAAC in situ library
+
+FFTW3
+"""""
+- required for Shadowgraphy plugin
+- *from tarball:*
+
+  - ``mkdir -p ~/src ~/lib``
+  - ``cd ~/src``
+  - ``wget -O fftw-3.3.10.tar.gz http://fftw.org/fftw-3.3.10.tar.gz``
+  - ``./configure --prefix="$FFTW_ROOT"``
+  - ``make``
+  - ``make install``
+- *environment:* (assumes install from source in ``$HOME/lib/fftw-3.3.10``)
+
+  - ``export FFTW3_ROOT =$HOME/lib/fftw-3.3.10
```

Comment:
These lines are lines that should be added to the picongpu profile and they are not directly relevant for the install process? It's done equivalent here to how it's done for blosc or pngwrite above

Reply:
I just thought that since the install process uses the variable `$FFTW3_ROOT`, it shoukld be defined before, but you are right, we usually do it later. Thus, fine withe me.